test_conformance/relationals/test_shuffles.cpp

```diff
@@ -217,8 +217,10 @@ const char *get_order_string( ShuffleOrder &order, size_t vecSize, cl_uint lengt
         byNumber = (useNumbersFlip++)%2;
     }
     // Do not use xyzw for vectors whose length is not 2 or 4 per the spec.
-    if (vecSize != 2 || vecSize != 4 || vecSize != 3)
+    if (vecSize != 2 && vecSize != 4 && vecSize != 3)
```

Comment:
Would say that `if (vecSize < 2 && vecSize > 4)` is slightly more readable, but it's a bit of a personal preference.

Reply:
I went for the minimal fix; would have applied your suggestion but following Ben's comment I believe we can drop this altogether.